script/playbackstrategy/modifiers/cehtml.js

```diff
@@ -0,0 +1,779 @@
+/**
+ * @fileOverview Requirejs module containing device modifier for CEHTML media playback
+ * @preserve Copyright (c) 2013-present British Broadcasting Corporation. All rights reserved.
+ * @license See https://github.com/bbc/tal/blob/master/LICENSE for full licence
+ */
+
+define(
+    'bigscreenplayer/playbackstrategy/modifiers/cehtml',
+  [
+    'bigscreenplayer/playbackstrategy/modifiers/mediaplayerbase',
+    'bigscreenplayer/debugger/debugtool'
+  ],
+    function (MediaPlayerBase, DebugTool) {
+      'use strict';
+      /**
+       * Main MediaPlayer implementation for CEHTML devices.
+       * Use this device modifier if a device implements the CEHTML media playback standard.
+       * It must support creation of &lt;object&gt; elements for media mime types, and those objects must expose an
+       * API in accordance with the CEHTML specification.
+       * @name antie.devices.mediaplayer.CEHTML
+       * @class
+       * @extends antie.devices.mediaplayer.MediaPlayer
```

Comment:
Should probably remove this stuff. Same with the license stuff referencing TAL at top of file.

Reply:
All removed!